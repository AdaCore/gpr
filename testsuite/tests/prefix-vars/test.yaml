description:
    test variable reference with project.package.name
driver: build_and_run
project_file: prefix_vars
main: main
