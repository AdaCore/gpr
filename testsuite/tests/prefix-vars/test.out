Demo     K_STANDARD
   Compiler'Local_Configuration_Pragmas -> /toto
